src/overtone/studio/inst.clj

```diff
@@ -81,16 +110,23 @@
   "Control the volume of a single instrument."
   [inst vol]
   (ensure-node-active! inst)
-  (ctl (:mixer inst) :volume vol)
+  (ctl @(:mixer inst) :volume vol)
   (reset! (:volume inst) vol))
 
 (defn inst-pan!
   "Control the pan setting of a single instrument."
   [inst pan]
   (ensure-node-active! inst)
-  (ctl (:mixer inst) :pan pan)
+  (ctl @(:mixer inst) :pan pan)
   (reset! (:pan inst) pan))
 
+(defn inst-mixer-ctl!
+  "Control a named parameters of the output mixer of a single instrument."
+  [inst & args]
```

Comment:
Couldn't `args` be a map instead of kv args? It's more composable to have hash maps (but yeah, I know we use this pattern in a lot of places, it just doesn't seem very nice)

Reply:
How about both? Use `[inst & {:as args}]` and then `(apply ctl @(:mixer inst) (-> args seq flatten))`?